src/Serializers/ContractBytecodeSerializer.js

```diff
@@ -0,0 +1,35 @@
+const RLP = require('rlp')
+const BaseSerializer = require('./BaseSerializer')
+const BytecodeSerializer = require('./BytecodeSerializer')
+const IntSerializer = require('./IntSerializer')
+const {byteArray2Int, byteArray2Hex} = require('../utils/int2ByteArray')
+
+class ContractBytecodeSerializer extends BaseSerializer {
+    constructor(globalSerializer) {
+        super()
+        this._bytecodeSerializer = new BytecodeSerializer(globalSerializer)
```

Comment:
the global serializer can be used to get rid of the specific deps here.

Reply:
As I see, `BytecodeSerializer` is not registered in Serializer, so I can't call it using the global serializer's `deserializeWithType` method.

Reply:
Correct, I've overlooked that. There is no such type tag.